CommonVoice-Data/data/gutenberg/20949.txt

```diff
@@ -1 +0,0 @@
-On a conservé à l'identique la ponctuation et l'usage des majuscules
```

Comment:
it's  not empty

Reply:
It's one line. It's not a book. It's not significant nor representative. It's clearly buggy. It's part of the tail of `ls -l`. Should I really argue?
(And by the way, it's old french: http://www.gutenberg.org/files/20949/20949-h/20949-h.htm)

Reply:
It's still a valid sentence that has no problem at all.

Reply:
Let's keep it simple.
Do you consider this **file**'s presence in the repository (given it's current one-line content) as completely valid?

Reply:
yes, even if it looks weird